nixos/desktop/default.nix

```diff
@@ -237,7 +231,7 @@ in
   );
 
   environment.variables = {
-    VISUAL = "${lib.getExe zed-fhs} --wait";
+    VISUAL = "${lib.getExe edge-pkgs.zed-editor} --wait";
```

Comment:
https://github.com/kachick/dotfiles/issues/755 again. It is frustrated in these changes...